maldoca/service/common/process_doc.cc

```diff
@@ -299,7 +300,17 @@ absl::Status DocProcessor::ProcessDoc(absl::string_view file_name,
     }
   }
 
-  MALDOCA_RETURN_IF_ERROR(current_document_pipeline->Process(doc));
+  ::maldoca::ResetFailedEncoding();
```

Comment:
maybe we should do this in BufferToUtf8::Init() so it's always referring to the last try.

Reply:
done